internal/mkcw/embed/check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
nit: scripts tend to use `#!/usr/bin/env bash` here

https://github.com/containers/buildah/commit/b288b7a3d96a0c8ee2e0d580402a1a335a29f35b